packages/fxa-auth-server/lib/routes/utils/signup.js

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const P = require('../../promise');
+
+module.exports = (log, db, mailer, push, verificationReminders) => {
```

Comment:
If this is important enough to pull out to a separate module, should it also have some dedicated tests that exercise it directly? I'm wary of how thorough the test coverage is when it's one-layer removed.

And it makes auditing test coverage harder too, because we have to go looking in a completely different place to see if things have tests. And if want to make changes to this module later, it's much harder to test them by writing/changing endpoint tests than it is direct ones.

(kind of related to my testing OKR / #2087)

Reply:
Lets make a dent on that issue! I'm good with adding a few tests for this. We can probably expand on it once we update our test coverage tool and see where the gaps are.